addons/l10n_ec/migrations/3.5/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, SUPERUSER_ID
+
+def update_withhold_income_sale_repartition_lines(env):
+    # Delete tag_ids for withhold income sale repartition lines
+    tax_group_id = env.ref('l10n_ec.tax_group_withhold_income_sale')
+    repartition_line_ids = env['account.tax.repartition.line'].search([('invoice_tax_id.tax_group_id', '=', tax_group_id.id)])
+    repartition_line_ids.write({
+        'plus_report_expression_ids': False,
+        'minus_report_expression_ids': False,
```

Comment:
@stevTresCloud let's identify the exact tag first, you never know what people might have done